src/main/java/io/codelex/enums/practice/Exercise2/SPS.java

```diff
@@ -1,7 +1,21 @@
 package io.codelex.enums.practice.Exercise2;
 
+import java.util.Random;
+
 public enum SPS {
     SCISSORS,
     PAPER,
-    STONE
+    STONE;
+
+    public static SPS computersChoice() {
+        Random random = new Random();
+        int randomNum = random.nextInt(3);
+        SPS randomSPSValue = null;
```

Comment:
just return element
un lai izvairītos no out of bound, priekš random varētu padot dinamisku enum skaitu